ui/packages/shared/profile/src/ProfileIcicleGraph/index.tsx

```diff
@@ -156,16 +138,15 @@ const ShowHideLegendButton = ({
   );
 };
 
-const GroupAndSortActionButtons = ({navigateTo}: {navigateTo?: NavigateFunction}): JSX.Element => {
-  const [storeSortBy = FIELD_FUNCTION_NAME, setStoreSortBy] = useURLState({
-    param: 'sort_by',
-    navigateTo,
+const GroupAndSortActionButtons = (): JSX.Element => {
+  const [storeSortBy, setStoreSortBy] = useURLState('sort_by', {
+    defaultValue: FIELD_FUNCTION_NAME,
   });
   const {compareMode} = useProfileViewContext();
 
-  const [storeGroupBy = [FIELD_FUNCTION_NAME], setStoreGroupBy] = useURLState({
-    param: 'group_by',
-    navigateTo,
+  const [groupBy, setStoreGroupBy] = useURLState<string[]>('group_by', {
+    defaultValue: [FIELD_FUNCTION_NAME],
+    alwaysReturnArray: true,
   });
```

Comment:
what's the difference between setting a default value like this i.e. using the `defaultValue` prop,
```suggestion
  const [groupBy, setStoreGroupBy] = useURLState<string[]>('group_by', {
    defaultValue: [FIELD_FUNCTION_NAME],
    alwaysReturnArray: true,
  });
```

and also doing it like the one a bit further in this file, like this:
```suggestion
  const [compareAbsolute = compareAbsoluteDefault, setCompareAbsolute] =
    useURLState('compare_absolute');
```

Reply:
No technical difference, just an explicit way to set the default value right now. 

But in the future, I'm thinking we should move away from this distributed (passing it while accessing the hook) default value usage to a more centralized usage by passing the default values in the `URLStateProvider`. So that the consuming hooks don't have to worry about the default state and also when a param is in the default state, we don't actually have to push it to the URL. 
This will also help keep the URL to be a bit concise by only having non-default values.